### PR TITLE
JavaScript Capstone Project - Delicious Meals

### PR DESCRIPTION
**Pull request by BHObserver**
### In this branch-

- **We have merged our feature branches**

### In our feature branches-

- **We have created the basic structure for the website and designed it**
- **We have created the pop up for the comment**
- **We have used involvement API to save to and fetch the data**
- **We have created the counter for both likes and comments**

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+  background-color: aliceblue;
+  margin: 0 auto;
+  max-width: 920px;
+  width: 50vw;
+  height: 70%;
+  border-radius: 8px;
+  margin-top: 8vw;
+  overflow-y: auto;
+  overflow-x: hidden;
+}
```
Comment (Alusp) [inline src/style.css]: - [ ] A very good job so far, :clap: , however, Please ensure that your popup design matches the given requrement by increasing the height of your `popup` to something like "80%" or "85%", as it is right now the image in your popup takes up the whole popup window. Matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 
- [ ] And Please ensure that the order of the design looks like the given one, the comment form and its button should be at the bottom of the popup window while displaying the comments above the form

Below is an illustration of yours and the given Design :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

Student Popup1

</td>
      <td>

Comments

</td>
    </tr>
  </table>

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @BHObserver and @Umuhan, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
close

 ## Highlights

- [ ] Linters set-up ✔️
- [ ] Commit messages descrptive ✔️
- [ ] Professional Github Flow used ✔️
- [ ] Professional PR Summary Flow used ✔️
- [ ] UI Closer to the given design :heavy_check_mark:  
- [ ] Use ES6 syntax.
- [ ] Use webpack.
- [ ] Send and receive data from an API.
- [ ] Write units tests for a JavaScript app.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alusp in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 1 above)
Comment (Umuhan) [reply src/style.css]: Thanks. Done.

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @BHObserver  and @Umuhan ,
# STATUS: APPROVED! 🥇 💯 
done

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**